### PR TITLE
Cleanup eddb plugin

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**
Usual string of PEP8 and readability improvements

### CODE REVIEW

File: plugins/eddb.py [hunk 1]
```diff
@@ -24,55 +24,64 @@
...
-this.system_address = None
-this.system_population = None
-this.station_link = None
-this.station = None
-this.station_marketid = None
+this.system_link: Optional[str] = None
+this.system: Optional[str] = None
+this.system_address: Optional[str] = None
+this.system_population: Optional[int] = None
+this.station_link = None  # tk thing, not annotated
```
[the remark this answers is not in the document]
Comment (A-UNDERSCORE-D) [reply plugins/eddb.py:42]: was not sure what it was and was half asleep enough to not want to go digging to figure it out. I'll poke at that today

**Comment by A-UNDERSCORE-D** [comment]
This needs a rebase

File: plugins/eddb.py [hunk 2]
```diff
@@ -24,55 +24,68 @@
 
 
 import sys
+import tkinter
```
Comment (Athanasius) [inline plugins/eddb.py:27]: Is this necessary given the conditional 'from' import below ?

Comment (A-UNDERSCORE-D) [reply plugins/eddb.py]: removed -- was added in an auto import while I was typing

**Pull request merged**